MCprep_addon/materials/prep.py

```diff
@@ -308,10 +296,35 @@ def execute(self, context):
 		addon_prefs = util.get_user_preferences(context)
 		self.track_param = context.scene.render.engine
 		self.track_exporter = addon_prefs.MCprep_exporter_type
+		return 0
+
+
+class MCPREP_OT_prep_materials(McprepPrepMaterial, Operator):
+	"""Fixes materials and textures on selected objects for Minecraft rendering"""
+	bl_idname = "mcprep.prep_materials"
+	bl_label = "MCprep Materials"
+	bl_options = {'REGISTER', 'UNDO'}
+
+	skipUsage: BoolProperty(
+		default=False,
+		options={'HIDDEN'})
+
+	def invoke(self, context, event):
+		return context.window_manager.invoke_props_dialog(
+			self, width=300 * util.ui_scale())
+
+	track_function = "materials"
+	track_param = None
+	track_exporter = None
+	@tracking.report_error
+	def execute(self, context):
+		result = self.prep_mat(context)
+		if result:
+			return {'CANCELLED'}
```

Comment:
I don't see the point in taking a perfectly good operator and move the body of `execute` to a separate method only to call the method in `execute`.

Reply:
I actually think the intent here is good: we don't want to be calling duplicate code from different places, and if we can reuse the self.prep_mat function as-is in 3 places vs rewriting the same boilerplate, then it's probably a good thing. I'm not convinced as of yet it's a great idea to force the same popup UI for each place we are including prep materials options. I'd have to try it live first to see how it's looking and feeling. There's some pros and cons here since we're also clobbering together any warning messages or popups that happen in the normal course of prep materials, but then again maybe those would be still relevant anyways.

So, for the re-opened PR, feel free to keep this same original structure as a starting point, and we can go from there.